src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java

```diff
@@ -15,7 +15,7 @@ public class DaaMapper implements RowMapper<DataAccessAgreement>, RowMapperHelpe
   @Override
   public DataAccessAgreement map(ResultSet resultSet, StatementContext statementContext) throws SQLException {
     DataAccessAgreement daa;
-    Integer daaId = hasColumn(resultSet, "daa_id") ? resultSet.getInt("daa_id") : null;
+    Integer daaId = hasColumn(resultSet, "daa_id") && resultSet.getInt("daa_id") > 0 ? resultSet.getInt("daa_id") : null;
```

Comment:
Suggestion: pull this out into a `hasNonZeroColumn` method that these mappers can use.

Reply:
+1 this suggestion.

Reply:
Great idea, thanks!